src/TitleDetails/AddItemPlugin/AddItemPlugin.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Pluggable,
+  stripesShape,
+  withStripes,
+} from '@folio/stripes/core';
+
+const AddItemPlugin = ({
+  addItem,
+  disabledButtonProps,
+  instanceId,
+  isAddItemButtonDisabled,
+  locationId,
+  stripes,
+}) => {
+  return (
+    <Pluggable
+      addItem={addItem}
+      aria-haspopup="true"
+      disabled={isAddItemButtonDisabled}
+      instanceId={instanceId}
+      locationId={locationId}
+      searchButtonStyle="default"
+      searchLabel={<FormattedMessage id="ui-receiving.piece.actions.addItem" />}
+      stripes={stripes}
```

Comment:
IMHO, it makes not too much sense to pass stripes there, and it's better to use withStripes or even stripesConnect inside the plugin

Reply:
prepared [PR for ui-plugin-create-item](https://github.com/folio-org/ui-plugin-create-item/pull/27)